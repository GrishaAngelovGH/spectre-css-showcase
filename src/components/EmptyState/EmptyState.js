const EmptyState = () => (
  <div className='columns'>
    <div className='column col-4 col-mx-auto'>
      <div className='empty'>
        <div className='empty-icon'><i className='icon icon-3x icon-mail'></i></div>
        <p className='empty-title h5'>You've successfully signed up</p>
        <p className='empty-subtitle'>Click the button to invite your friends</p>
        <div className='empty-action'>
          <button className='btn btn-primary'>Invite your friends</button>
        </div>
        <div className='empty-action'>
          <button className='btn btn-link'>Skip</button>
        </div>
      </div>
    </div>
  </div>
)

export default EmptyState