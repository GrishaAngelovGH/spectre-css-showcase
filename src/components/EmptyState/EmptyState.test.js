import { render } from '@testing-library/react'

import EmptyState from './EmptyState'

test('should render EmptyState component', () => {
  const view = render(<EmptyState />)

  expect(view).toMatchSnapshot()
})