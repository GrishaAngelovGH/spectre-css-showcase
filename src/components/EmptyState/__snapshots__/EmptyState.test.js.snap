// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EmptyState component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="columns"
      >
        <div
          class="column col-4 col-mx-auto"
        >
          <div
            class="empty"
          >
            <div
              class="empty-icon"
            >
              <i
                class="icon icon-3x icon-mail"
              />
            </div>
            <p
              class="empty-title h5"
            >
              You've successfully signed up
            </p>
            <p
              class="empty-subtitle"
            >
              Click the button to invite your friends
            </p>
            <div
              class="empty-action"
            >
              <button
                class="btn btn-primary"
              >
                Invite your friends
              </button>
            </div>
            <div
              class="empty-action"
            >
              <button
                class="btn btn-link"
              >
                Skip
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="columns"
    >
      <div
        class="column col-4 col-mx-auto"
      >
        <div
          class="empty"
        >
          <div
            class="empty-icon"
          >
            <i
              class="icon icon-3x icon-mail"
            />
          </div>
          <p
            class="empty-title h5"
          >
            You've successfully signed up
          </p>
          <p
            class="empty-subtitle"
          >
            Click the button to invite your friends
          </p>
          <div
            class="empty-action"
          >
            <button
              class="btn btn-primary"
            >
              Invite your friends
            </button>
          </div>
          <div
            class="empty-action"
          >
            <button
              class="btn btn-link"
            >
              Skip
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
